### DISCUSSION TITLE
Automatically set last cooked when meal is added to meal plan

### DISCUSSION THREAD

**Discussion by anytimesoon** [TandoorRecipes/recipes#3398] [Q&A]
I would like to know how I can set it up so that once a meal has been added to the meal plan and the day has past, the last cooked date is updated on that recipe.

I'm pretty sure this feature either exists or has existed, but it's not currently working for me, and I can't find a setting for it anywhere.

**Comment by smilerz** [comment]
You have to log cooking.

**Reply by anytimesoon** [reply]
Thank you for your reply. I've just had another look through the settings, but I can't find it 🤷‍♂️

**Reply by smilerz** [reply]
![image](https://github.com/user-attachments/assets/fc7da662-1a03-40f1-be13-87a58bd05149)

**Reply by anytimesoon** [reply]
This is a manual process each time, right? I thought there was a way for it to automatically be logged from the meal plan.

Something like: I add a recipe to the meal plan for Monday and on Tuesday, it gets logged as having been cooked on Monday.

**Reply by smilerz** [reply]
Yes. It's manual.  Being on the meal plan doesn't necessarily mean it got cooked.

**Reply by anytimesoon** [reply]
That's true, but it is for about 99% of the time for me. It's not a huge thing, just would have been nice.

Thank you for your help!

**Reply by sortab** [reply]
> That's true, but it is for about 99% of the time for me. It's not a huge thing, just would have been nice.
> 
> Thank you for your help!

I'd also wish for this feature to be possible and not having to manualle log a cooking.
